### DISCUSSION TITLE
Cannot login after initail setup

### DISCUSSION THREAD

**Discussion by starworld99** [nextcloud/all-in-one#4351] [Questions] [answered]
installed nextcloud/all-in-one:latest on my QNAP TS-466, the installation successfully, but after that when I tried to login with initial credentails, the login failed, I even copied the password from /var/lib/docker/volume/nextcloud-aio-mastercontainer/-data/data/configration.json, nothing help.

Could someone help me to login, thanks in advance.

**Comment by szaimen** [comment]
Hi, can you show a screenshot of what happens if you cannot log in?

**Reply by starworld99** [reply]
Actually, no message returned to user, the login page just back to the password input screen.

The url is http://a.b.c.d:11000/login?direct=1&user=admin

**Comment by szaimen** [comment, marked as answer]
Hi, you use the wrong url to log in. Please try via `https://your-nc-domain.com`
